### ISSUE TITLE
Q: Stickler for the Rules: Can a guest break multiple rules at once?

### ISSUE THREAD

**Issue by mark-oli-mts** [RoboCupAtHome/RuleBook#602]
The current rulebook only states that "three of the guests are breaking rules". Is it possible that a guest breaks more than one rule?

**Comment by kyordhel** [comment]
# TL;DR; Yes, she can.

Very interesting question you're making there. I'd say yes, it is possible (although highly unlikely) that a guest breaks more than one rule. Explanation below.

Robots should be able to realize that many thing can go wrong at the same time. Let's say I'm a burglar and just entered through the window, so I'm wearing shoes in the forbidden room, and have no drink at all. A decent robot should call the police after realizing the shattered window and my crowbar, but for @Home purposes, the robot will just force me to take off my shoes, leave the forbidden room, and take my drink, so I spend the rest of my life in that mental asylum.

Being that said, and considering that 1) this is the first year we run this test, and 2) that awareness and activity recognition is a challenging topic beyond state-of-the-art research; I'd keep it simple for now with the following constraints:
1. A person can break more than one rule only if there are not sufficient volunteers available.
2. When a person breaks more than one room, it most likely not be at once, and not in the same room unless impossible otherwise (e.g. take off and put on shoes while switching rooms).
3. The robot is expected to recognize only one rule being broken (if you catch more, awesome! You save time).

This means I can be in the bedroom (forbidden room) without drink, and then move to the kitchen. In the bedroom the robot must scold me for being there and, optionally, for not having a drink. In the kitchen the robot must scold me for not having a drink only.

The case of double detection would be discussed during Team Leader meeting, but most likely would count as detecting two infractions (remember you can do up to six).

**Comment by mark-oli-mts** [comment]
Thanks for the quick answer!
